judge/models/contest.py

```diff
@@ -236,6 +255,7 @@ def time_before_start(self):
 
     @property
     def time_before_end(self):
+        # In terms on contest time itself
```

Comment:
We don't want to blowup the MCPT/WLMOJ to DMOJ/online-judge changes with comments that DMOJ does not have, so please remove these (same with some of the other comments). Feel free to PR these to DMOJ though.